src/client/resources/conversions.ts

```diff
@@ -116,7 +159,7 @@ export function createStorageSecret(
     apiVersion: 'v1',
     data: {
       'aws-access-key-id': encodedAccessKey,
-      'aws-secret-access-key': encodedSecretKey,
+      'aws-secret-access-key-id': encodedSecretKey,
```

Comment:
Need to look more to determine which value this should be.

'aws-secret-access-key' or 'aws-secret-access-key-id'.

Reply:
Addressed.